tests/testthat/test-auditbayesianplanning.R

```diff
@@ -183,12 +183,12 @@ test_that("Across Expected Errors (Current: 0) plot matches 3", {
 test_that("<b>Table 3.</b> Descriptive Statistics for Prior and Expected Posterior Distribution results match", {
 	table <- results[["results"]][["planningContainer"]][["collection"]][["planningContainer_priorAndPosteriorStatistics"]][["data"]]
 	jaspTools::expect_equal_tables(table,
-		list(9500, "beta-binomial(N = 9987, <unicode><unicode> = 1, <unicode><unicode> = 1)",
-			 0.200079992000814, 0.799920007999186, 523, 0.250125000000022,
-			 0.8977, "Prior", 1927, "beta-binomial(N = 9987, <unicode><unicode> = 1, <unicode><unicode> = 14)",
-			 0.955981093321683, 0.0440189066783169, 0, 21.7175110755894,
-			 0.1927, "Expected posterior", -7573, "", 4.77799445992478, 0.0550291357112318,
-			 -523, 86.8266309868567, "", "Expected shift"))
+		list(9501, "beta-binomial(N = 10000, <unicode><unicode> = 1, <unicode><unicode> = 1)",
+			 0.200079992000814, 0.799920007999186, "", 0.250125000000022,
+			 "", "Prior", 1926, "beta-binomial(N = 9987, <unicode><unicode> = 1, <unicode><unicode> = 14)",
+			 0.956181004359911, 0.0438189956400888, 0, 21.8211529130788,
+			 1926, "Expected posterior", -7575, "", 4.77899361549366, 0.054779221924567,
+			 "", 87.2409911567292, "", "Expected shift"))
```

Comment:
So were the old results wrong?

Reply:
Yes there was an error in the calculation for the beta-binomial mode.